Makefile

```diff
@@ -0,0 +1,20 @@
+include .env
+export
+
+up:
```

Comment:
Don't all of these need to take place in the submodule directory?

Reply:
Nope.  That's not necessary